### ISSUE TITLE
Hotkeys don't work outside of English layout

### ISSUE THREAD

**Issue by chekoopa** [bouncepaw/mycorrhiza#86]
**How to reproduce**: try pressing `h` on any hypha page or `Ctrl-B` in editor, when using Russian or another non-Latin keyboard layout.

I've localized it to `static/shortcuts.js`, as its code uses `key` property to retrieve the pressed key, however we also have layout-agnostic `code` property (and `which`, now deprecated). We can redo the code to use these values for hotkeys (convert `Key*` and `Digit*` keys, and maybe symbols, and cast everything else).

As an alternative, we could enable users to add their own shortcuts and partly circumvent the problem (also improving QoL for user JS plugins). Or do both proposed things.

That is actually a common problem in a variety of software (e.g. Vim, and even Github).

**Comment by bouncepaw** [comment]
Yeah, there is such a problem. I think it is not solvable because users may have weird keyboard layouts. That's why we kinda decided to ignore the problem for the time being. Perhaps @handlerug can save the day

**Comment by chekoopa** [comment]
> I think it is not solvable because users may have weird keyboard layouts. 

I think we should just open the hotkey list (i.e. bind it to `window`) for other plugins. It'll also enable werid layout users to bind and remap by their tastes.

**Comment by bouncepaw** [comment]
I think I want to include йцукен hotkeys too now.

**Comment by chekoopa** [comment]
I still see the better option in implementing these through `code` (as it would match pieces of UX outside)

**Comment by bouncepaw** [comment]
Yeah, it's probably good, I'll take a look.

**Comment by bouncepaw** [comment]
Not actually fixed, tested on Chrome on Windows.
